app/components/header_component/view.rb

```diff
@@ -2,19 +2,14 @@ module HeaderComponent
   class View < ViewComponent::Base
     attr_accessor :navigation_items, :phase_name
```

Comment:
can the navigation_items accessor be removed?

Reply:
Done!